src/modules/endpoint/reducers/pushNews.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import type { CityContentStateType } from '../../app/StateType'
+import type { PushNewsActionType } from '../../app/StoreActionType'
+
+const pushNews = (state: CityContentStateType, action: PushNewsActionType): CityContentStateType => {
+  const { newsList, path, key, language, resourceCache, cityLanguages, city, type, page, oldNewsList, hasMoreNews } = action.params
+
+  if (!key) {
+    throw new Error('You need to specify a key!')
+  }
+
+  const getNewsRoute = (): any => {
+    if (!path) {
+      const allAvailableLanguages = new Map(cityLanguages.map(language => [language.code, null]))
+      if (page && oldNewsList) {
+        return { status: 'ready', path: null, models: [...oldNewsList, ...newsList], hasMoreNews, allAvailableLanguages, language, city, type, page }
+      }
+      return { status: 'ready', path: null, models: newsList, allAvailableLanguages, language, city, type, page, hasMoreNews }
+    }
+    const newsItem: ?any = newsList.find(newsItem => `${newsItem.id}` === path)
+    if (!newsItem) {
```

Comment:
`News Item with path ${path} was not found in supplied models.` should be 
`News item with path ${path} was not found in supplied models.` with `item` in small letters